src/native/libs/System.Native/pal_networking.c

```diff
@@ -1795,6 +1795,12 @@ static bool TryGetPlatformSocketOption(int32_t socketOptionLevel, int32_t socket
                     return true;
 #endif
 
+#ifdef IP_DONTFRAG
+                case SocketOptionName_SO_IP_DONTFRAGMENT:
+                    *optName = IP_DONTFRAG;
```

Comment:
Will this work for IPv6, or does that case need to be handled by using `IPV6_DONTFRAG` instead?